src/main/java/com/lithium/dbi/rdbi/ratelimiter/RateLimiterConfiguration.java

```diff
@@ -0,0 +1,46 @@
+package com.lithium.dbi.rdbi.ratelimiter;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class RateLimiterConfiguration {
```

Comment:
this should stay in the project where it came from.  The individual values should be passed in as-needed.  I think the only thing you'll end up needing is the keyPrefix since the rest is used to construct the RDBI instance (which is pre-created and provided to the rate limiter factory anyway.